crates/iota-sdk-ffi/src/types/transaction.rs

```diff
@@ -227,19 +227,19 @@ impl EndOfEpochTransactionKind {
         Self(iota_types::EndOfEpochTransactionKind::AuthenticatorStateExpire(tx.0.clone()))
     }
 
-    #[uniffi::constructor]
-    pub fn bridge_state_create(chain_id: &CheckpointDigest) -> Self {
-        Self(iota_types::EndOfEpochTransactionKind::BridgeStateCreate {
-            chain_id: **chain_id,
-        })
-    }
-
-    #[uniffi::constructor]
-    pub fn bridge_committee_init(bridge_object_version: u64) -> Self {
-        Self(iota_types::EndOfEpochTransactionKind::BridgeCommitteeInit {
-            bridge_object_version,
-        })
-    }
+    // #[uniffi::constructor]
+    // pub fn bridge_state_create(chain_id: &CheckpointDigest) -> Self {
+    //     Self(iota_types::EndOfEpochTransactionKind::BridgeStateCreate {
+    //         chain_id: **chain_id,
+    //     })
+    // }
+
+    // #[uniffi::constructor]
+    // pub fn bridge_committee_init(bridge_object_version: u64) -> Self {
+    //     Self(iota_types::EndOfEpochTransactionKind::BridgeCommitteeInit {
+    //         bridge_object_version,
+    //     })
+    // }
```

Comment:
Should be just deleted, other PRs are also doing that already, time to merge one :trollface: